packages/frontend/src/components/CreateHouseModal.tsx

```diff
@@ -0,0 +1,118 @@
+import {
```

Comment:
Make a directory inside `components` for these. The intent is each page has a folder inside `components`, so for example, place these modals in `/components/account`